components/post-image/PostImage.tsx

```diff
@@ -37,12 +32,9 @@ const PostImage = ({ src, alt }: IProps) => {
               title: alt || 'Image no alt defined',
               fullWidth: true,
               content: (
-                <div className="w-full">
-                  <Image
-                    src={src}
-                    alt={alt}
-                    loadingType={EImageLoadingType.EAGER}
-                  />
+                <div className="global-width mx-auto">
+                  {/*aspect 17:8 (mumble image)*/}
+                  <ImageNext src={src} alt={alt} width={1280} height={602.35} />
```

Comment:
Is this the only way to implement the aspect-ratio? Is this a performance reason that you defined the specific width and height?

Reply:
The width and height are not the specific image sizes. this works more like a placeholder, so that layout shifts are avoided. still the aspect ratio is controlled by css.